driver/evrmemmap.h

```diff
@@ -97,12 +97,19 @@ struct DBufInfo {
 /* These must be powers of two!!! */
 #define MAX_EVR_EVTQ 1024
 #define MAX_EVR_DBQ     4
+#define MAX_EVR_DBQ2   16
 struct EvrQueues {
   struct FIFOEvent evtq[MAX_EVR_EVTQ];     /* 12K */
   struct DBufInfo  dbq[MAX_EVR_DBQ];       /*  8K + 16 */
   long long ewp;                           /*  20 */
   long long dwp;
   long long fifofull;
+  /*
+   * Sigh... turns out that a data buffer queue of 4 isn't deep enough.
+   * But let's be good members of the First Church of Backwards Compatibility
+   * and just add a *second* larger queue!  AMEN!
```

Comment:
What kinds of backwards compatibility problems can arise from doing the simpler solution of expanding the previous, smaller queue? I guess this file is called memmap but I'm not familiar with how it is used.

Reply:
The old event2 is compiled believing that the size is 4, and won't map the last 12 entries.

Reply:
How this is used: struct EvrQueues is what is memory-mapped into user space to read the events and data buffers instead of reading them directly from the hardware.  This is how we implement sharing... the driver reads the hardware on behalf of the connected user-space processes.

Reply:
There is a "dwp" index to the most recent data buffer.  We constantly increment this, but write the element modulo queue size.  So the old event2 uses 4... the new event2 will use 16 and the *second* larger queue.

Reply:
I read a bit more closely (and context shifted appropriately) and I largely understand now